integration-tests/camel/invoked/root/camel-debezium/pom.xml

```diff
@@ -10,6 +10,11 @@
 
     <artifactId>quarkus-universe-integration-tests-camel-debezium</artifactId>
 
+    <!-- TODO: Remove when all platform participants (i.e debezium) can work with testcontainers >= 1.15.0 -->
```

Comment:
Have you reported this issue to the Debezium team? /cc @gunnarmorling @Naros

Reply:
Seems that Debezium 1.4 aligns with testcontainers 1.15.x. So when that eventually makes it into platform, I can remove the workaround.

Reply:
OK, thanks!